src/tickit/adapters/interpreters/command/command_interpreter.py

```diff
@@ -83,8 +72,16 @@ async def handle(
                 for arg, argtype in zip(args, get_type_hints(method).values())
             )
             resp = await method(*args)
-            if not isinstance(resp, AsyncIterable):
-                resp = wrap_as_async_iterable(resp)
+            if not isinstance(resp, AsyncIterator):
+                resp = wrap_as_async_iterator(resp)
             return resp, command.interrupt
-        resp = CommandInterpreter.unknown_command()
-        return resp, False
+
+        msg = "Request does not match any known command"
+        # This is a pain but the message needs to match the input message
+        # TODO: Fix command interpreters' handling of bytes vs str
+        if isinstance(message, bytes):
+            resp = wrap_as_async_iterator(msg.encode("utf-8"))
+            return resp, False
+        else:
+            resp = wrap_as_async_iterator(msg)
+            return resp, False
```

Comment:
Duplication
```suggestion
        if isinstance(message, bytes):
            msg = msg.encode("utf-8")
        resp = wrap_as_async_iterator(msg)
        return resp, False
```

Reply:
That doesn't work. I faffed around with it for ages but this was the cleanest I could get it.

Reply:
Weird, but oh well